app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,27 @@
+package ar.com.wolox.android.example.ui.home
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+/**
+ * HomeActivity class
+ */
+class HomeActivity : WolmoActivity() {
+    override fun init() {
+        replaceFragment(R.id.vActivityBaseContent, HomeFragment.newInstance())
+    }
+
+    override fun layout(): Int {
```

Comment:
Use the kotlin sintax to reduce this expression to  ` override fun layout() = R.layout.activity_base`